### ISSUE TITLE
Test ELBO higher than train ELBO

### ISSUE THREAD

**Issue by ellimarip** [broadinstitute/CellBender#317]
I have used the raw feature-barcode matrix from https://support.10xgenomics.com/single-cell-gene-expression/datasets/2.1.0/pbmc4k and have ran CellBender with defaults. I am wondering why the test ELBO seems to be higher than train ELBO?

![learning_curve](https://github.com/broadinstitute/CellBender/assets/145563245/d58a5222-04e5-45e2-8b12-6598e3381da3)

**Comment by Ratoncito** [comment]
Yes I also have experienced this on some of my runs. Not sure how to interpret this. Perhaps this means that the pretrained model was better suited for the testing set than for the training set for some reason. Not sure why this is but am somewhat confidant this isn't a big deal/ isnt necessarily a bad thing.
<img width="1005" alt="image" src="https://github.com/broadinstitute/CellBender/assets/56166993/f0e153c6-e88b-44c1-ae9b-d0746f7da4d6">
